cards/multilang-product-standard/component.js

```diff
@@ -33,7 +33,7 @@ class multilang_product_standardCardComponent extends BaseCard['multilang-produc
       // text. A toggle will be supplied that can show or hide the truncated text.
       // Note: If you are using rich text for the details, you should not enable this feature.
       // showMoreDetails: {
-      //   showMoreLimit: 350, // Character count limit
+      //   truncatedDetails: profile.richTextDescription ? ANSWERS.formatRichText(profile.richTextDescription, 'richTextDescription', linkTarget, 350) : null, // The truncated rich text
```

Comment:
just double checking, do we want truncatedDetails for all cards? I see some cards like menuitem-standard and link-standard doesn't have this update (a quick search with '// Character count limit' shows 16 card files with showMoreLimit)